packages/uniforms-antd/__tests__/NumField.tsx

```diff
@@ -185,13 +185,13 @@ test('<NumField> - renders an InputNumber which correctly reacts on change (empt
   );
 
   expect(wrapper.find('input')).toHaveLength(1);
-  expect(
-    wrapper.find('input').simulate('change', { target: { value: '' } }),
-  ).toBeTruthy();
+
+  wrapper.find(InputNumber).props().onChange!('');
```

Comment:
I don't understand where is this coming from. Isn't the new version of `InputNumber` based on `input` as well?

Reply:
It is, but for some reason

```typescript
wrapper.find('input').simulate('change', { target: { value: '' } });
```

doesn't trigger `onChange`. Seems to be related to the [breaking changes](https://ant.design/changelog#4.13.0) in `antd@4.14.0`, although it's not explicitly stated there.

Reply:
OK, and could you check, whether or not the type fix I proposed is enough?

Reply:
Also, it looks like they updated their tests in the same way: https://github.com/ant-design/ant-design/pull/29373/files#diff-7fcfad35ca4d1e8f875d130aa8186b0157946e828e7ffb95908d6f928b3f6e1fR18. On the other hand, https://github.com/react-component/input-number/blob/c4ae6f89c21da3f746c6b40196015cc5c659b484/tests/input.test.tsx#L153-L158 would suggest, that we'd need to blur the input after the change.